templates/index.html

```diff
@@ -14,200 +14,221 @@
+          <strong>Note:</strong> MicroStack is in a beta state. We encourage you to test it, give us your <a class="p-link--external" href="https://bugs.launchpad.net/microstack">feedback</a> and <a class="p-link--external" href="https://discourse.juju.is/search?q=microstack">ask questions</a>. For information about commercial support for MicroStack, <a class="p-link--external" href="https://ubuntu.com/openstack/contact-us">contact Canonical</a>.
         </p>
       </div>
     </div>
     <div class="row">
-      <div class="col-8">
-        <h2>OpenStack in a snap</h2>
-        <h4>Zero-ops OpenStack on just about any Linux box.</h4>
+      <div class="col-8" style="margin-top: 2rem;">
+        <h2>Straightforward installation</h2>
+        <h4>Fully functional OpenStack in 2 commands and ~20 minutes.</h4>
       </div>
+      <pre class="p-heading--two"><code><span style="display: block;" class="typer" id="main" data-words="$ sudo snap install microstack --beta --devmode, $ sudo microstack init --auto --control" data-delay="50" data-deleteDelay="5000" data-colors="#111111"></span></code></pre>
+      <p>
+        <a class="p-link--external" href="https://ubuntu.com/tutorials/microstack-get-started#1-overview">MicroStack tutorial</a>
+      </p>
     </div>
+  </section>
+  <section class="p-strip--light">
     <div class="row u-equal-height">
-      <div class="col-4">
-        <ul class="p-list--divided u-no-margin--bottom__medium">
-          <li class="p-list__item is-ticked">Keystone</li>
-          <li class="p-list__item is-ticked">Glance</li>
-          <li class="p-list__item is-ticked last-item">Nova</li>
-        </ul>
+      <div class="col-4 p-divider__block u-vertically-center">
+        <h3 class="u-align--center">7 contributors</h3>
       </div>
-      <div class="col-4">
-        <ul class="p-list--divided u-no-margin--bottom__medium">
-          <li class="p-list__item is-ticked">Neutron with OVN</li>
-          <li class="p-list__item is-ticked">Cinder</li>
-          <li class="p-list__item is-ticked last-item">Dashboard</li>
+      <div class="col-4 p-divider__block u-vertically-center">
+        <h3 class="u-align--center" >~250 commits</h3>
       </div>
-      <div class="col-4">
-        <ul class="p-list--divided">
-          <li class="p-list__item is-ticked">Metrics</li>
-          <li class="p-list__item is-ticked">GPGPU bindings</li>
-          <li class="p-list__item is-ticked last-item">Clustering</li>
-        </ul>
+      <div class="col-4 p-divider__block u-vertically-center">
+        <h3 class="u-align--center" style="margin-left: 1.5rem; margin-right: 1.5rem;">~1,500 active installations</h3>
       </div>
     </div>
+  </section>
+  <section class="p-strip">
+      <div class="row">
+        <div class="col-6">
+          <h2>OpenStack for the edge</h2>
+          <h4>Comprehensive cloud platform for distributed low-latency applications</h4>
```

Comment:
Same here with the heading hierarchy, shouldn't be h2 then h4 (could you check the page for any of the same and replace the h4 with `<h3 class="p-heading--4">`)